api/envoy/api/v2/auth/cert.proto

```diff
@@ -315,9 +324,11 @@ message SdsSecretConfig {
 message Secret {
   // Name (FQDN, UUID, SPKI, SHA256, etc.) by which the secret can be uniquely referred to.
   string name = 1;
+
   oneof type {
     TlsCertificate tls_certificate = 2;
     TlsSessionTicketKeys session_ticket_keys = 3;
     CertificateValidationContext validation_context = 4;
+    core.DataSource trusted_ca = 5;
```

Comment:
I feel it is too bare to use DataSource directly.   It is better to wrap it with another message.

message TrustedCA {
  core.DataSource trusted_ca = 1;
};

In the future, it is easier to extend it.